### PR TITLE
Add dark mode to dashboard

### PR DESCRIPTION
**Pull request by krzysztoff1**
Screenshot 2023-03-04 at 20 59 34
Screenshot 2023-03-04 at 21 00 55

### CODE REVIEW

File: src/hooks/useTheme.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useEffect } from "react";
+
+import { useLocalStorage } from "./useLocalStorage";
+
+export const useTheme = () => {
+  const [isDarkMode, setIsDarkMode] = useLocalStorage<"true" | "false">(
```
Comment (Bartek532) [inline src/hooks/useTheme.ts:6]: I believe that we should store "theme" value in local storage not just if it's dark or not. Btw, we should also handle system theme by default

**Review by Bartek532 — COMMENTED** [review]
Could you attach some screenshots with app dark mode design? :)

File: src/hooks/useTheme.ts (hunk 1 above)
Comment (krzysztoff1) [reply src/hooks/useTheme.ts:6]: Good point, also maybe we should use dropdown for the theme in settings. By default, it would be set to `system`.

**Review by Bartek532 — APPROVED** [review]
Could you attach some screenshots with app dark mode design? :)

**Pull request merged**